src/lib.rs

```diff
@@ -227,6 +227,11 @@ where
         self.len
     }
 
+    /// How many bits does each packed element consume?
```

Comment:
Since I was also planning on making a PR for this (I just hadn't pushed it yet), I still have a doc string we could use:
```rust
    /// Return this `PackedVec`'s entry size in bits.
    ///
    /// # Example
    /// ```
    /// use packedvec::PackedVec;
    /// let v: Vec<u16> = vec![1, 2, 3, 4];
    /// let packedvec = PackedVec::new(v);
    /// assert_eq!(packedvec.bwidth(), 2);
    ///
    /// let v: Vec<u16> = vec![1, 8];
    /// let packedvec = PackedVec::new(v);
    /// assert_eq!(packedvec.bwidth(), 3);
    /// ```
```

Reply:
Good idea. I've pushed a commit based on this.